examples/seismic/model.py

```diff
@@ -349,12 +349,19 @@ def demo_model(preset, **kwargs):
 
 
 def initialize_damp(damp, nbpml, spacing, mask=False):
-    """Initialise damping field with an absorbing PML layer.
-
-    :param damp: The :class:`Function` for the damping field.
-    :param nbpml: Number of points in the damping layer.
-    :param spacing: Grid spacing coefficient.
-    :param mask: whether the dampening is a mask or layer.
+    """
+    Initialise damping field with an absorbing PML layer.
+
+    Parameters
+    ----------
+    damp : Function
+        for the damping field.
```

Comment:
"The damping field for absorbing boundary conditions."

Reply:
fixed everywhere